vendor/bat-native-ads/src/bat/ads/internal/resources/contextual/text_embedding/text_embedding_resource.cc

```diff
@@ -0,0 +1,68 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/resources/contextual/text_embedding/text_embedding_resource.h"
+
+#include <utility>
+
+#include "base/bind.h"
+#include "bat/ads/internal/base/logging_util.h"
+#include "bat/ads/internal/features/text_embedding_features.h"
+#include "bat/ads/internal/ml/pipeline/text_processing/embedding_processing.h"
+#include "bat/ads/internal/resources/resources_util_impl.h"
+#include "brave/components/l10n/common/locale_util.h"
+
+namespace ads {
+namespace resource {
+
+namespace {
+constexpr char kResourceId[] = "wtpwsrqtjxmfdwaymauprezkunxprysm";
+}  // namespace
+
+TextEmbedding::TextEmbedding()
+    : embedding_processing_pipeline_(
```

Comment:
`embedding_processing_` to match class name

Reply:
Updated in relevant places